packages/create-llama/helpers/llama-pack.ts

```diff
@@ -1,29 +1,70 @@
 import fs from "fs/promises";
+import got from "got";
```

Comment:
@thucpn looking good, but i think you also must change the generated python code. Llamapacks are now used like this:

<img width="750" alt="image" src="https://github.com/run-llama/LlamaIndexTS/assets/17126/60e441fb-7859-43e9-91e9-56dfe1334458">

see https://blog.llamaindex.ai/llamaindex-v0-10-838e735948f8 for details.

I guess you might need to modify the poetry project - Lee also worked on this - he can help